_etc/systemd/system/cvesearch.db_updater@cpe.service.d/redis-cache.conf

```diff
@@ -0,0 +1,11 @@
+# /etc/systemd/system/cvesearch.db_updater@cpe.service.d/redis-cache.conf
+# Redis caching is optional — if this drop-in is not present, CVE-Search
+# database updates for 'cpe' run without Redis cache.
```

Comment:
Updating the Redis cache every time the CPE database is updated may be excessive, particularly when there are few or no updates. Since the general timer runs almost hourly, it would be more efficient to introduce a separate timer that triggers the CPE update with Redis cache, perhaps once a day.